packages/commonwealth/client/scripts/views/pages/view_thread/ViewThreadPage.tsx

```diff
@@ -126,6 +158,14 @@ const ViewThreadPage = ({ identifier }: ViewThreadPageProps) => {
     threadId: parseInt(threadId),
   });
 
+  useEffect(() => {
+    app.proposalEmitter.on('redraw', forceRerender);
+
+    return () => {
+      app.proposalEmitter.removeAllListeners();
```

Comment:
do you use `removeAllListeners` on purpose? It removes also those listeners that were added in other files/components etc. Usually we use `off`. Just want to make sure...

Reply:
Hmm I think this was added in during a merge conflict. I think it can be removed so I have went ahead and removed it.